app/models/task.rb

```diff
@@ -264,6 +264,34 @@ def do_hold(last_check)
     end
   end
 
+  def name
+    return "Turn On" if self.type == "TurnOnTask"
+    return "Turn Off" if self.type == "TurnOffTask"
+    return "Run" if self.type == "RunThermostatTask"
+  end
+
+  def percent_ramp
```

Comment:
Oh sorry, I'll get rid of that. I was trying to see if I could incorporate the ramping period into the graph but at the moment, I don't think google graphs has a good feature to show the ramping stage.